.gitpod.yml

```diff
@@ -4,16 +4,17 @@ image:
 tasks:
   - name: Run PetClinic app
     init: ./mvnw -B dependency:go-offline package -DskipTests
```

Comment:
🤔 What is the average build time for this project, it would be bad UX if it takes too long (more than ~5 mins)

Reply:
This is a demo of prebuilds, so it's meant to be terribly long